File: P5/Civitas/src/civitas/OperacionInmobiliaria.java
/**
 * @author Salva
 */
package civitas;

/**
 *
 * @author abelrios
 */
public class OperacionInmobiliaria {

    private GestionesInmobiliarias gestion;
    private int numPropiedad;

    public OperacionInmobiliaria(GestionesInmobiliarias g, int n) {
        gestion = g;
        numPropiedad = n;
    }

    public OperacionInmobiliaria() {
        numPropiedad = 0;
    }

    public GestionesInmobiliarias getGestion() {
        return gestion;
    }

    public int getIndice() {
        return numPropiedad;
    }

}

File: P3/civitasruby/lib/civitas/operacion_inmobiliaria.rb
#encoding:utf-8

#To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

module Civitas
class Operacion_inmobiliaria
  attr_reader :num_propiedad, :gestion
  def initialize(gest,ip)
    @num_propiedad = ip
    @gestion = gest
  end
end
end
